### DISCUSSION TITLE
Running the Spark Demo on Microsoft Fabric / Synapse

### DISCUSSION THREAD

**Discussion by mattjbishop** [moj-analytical-services/splink#2037] [Show and tell]
Hi. For anyone wanting to try Splink on [Microsoft Fabric](https://learn.microsoft.com/en-us/fabric/) (including the trial capacity), I've uploaded a notebook containing a version of the [deduplicate_1k_synthetic](https://github.com/moj-analytical-services/splink/blob/master/docs/demos/examples/spark/deduplicate_1k_synthetic.ipynb) example adjusted for Fabric. The repo includes notes on how to set up a [Fabric Environment](https://learn.microsoft.com/en-us/fabric/data-engineering/create-and-use-environment) for Splink. 

You can find it here: [https://github.com/mattjbishop/splink-on-fabric](https://github.com/mattjbishop/splink-on-fabric)

**Comment by RobinL** [comment]
Thank you!
